### PR TITLE
[#861] Move feedback aggregating to different microservice

### LINKED ISSUE #861: Move feedback gathering from Lua code to microservice

### PR DESCRIPTION
**Pull request by kirillmakhonin**
This closes #861 
I've removed part of Lua code and implemented it in microservice on Golang.

### CODE REVIEW

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by kirillmakhonin** [comment]
bot, test app

**Review by mcdoker18 — DISMISSED** [review]

File: helms/legion/templates/edge/edge-server.yaml [hunk 1]
```diff
@@ -42,9 +42,13 @@ spec:
           value: "127.0.0.1"
         - name: FLUENTD_PORT
           value: "24224"
+        - name: FEEDBACK_COLLECTOR_BACKEND
```
Comment (alexander-semenets) [inline helms/legion/templates/edge/edge-server.yaml]: can we use a common name for feedback collector\aggregator since this is a single entity?

Comment (kirillmakhonin) [reply helms/legion/templates/edge/edge-server.yaml]: done

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by kirillmakhonin** [comment]
bot, test app

**Review by mcdoker18 — DISMISSED** [review]

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by kirillmakhonin** [comment]
bot, test app

**Comment by kirillmakhonin** [comment]
bot, test app

**Review by mcdoker18 — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -103,6 +107,14 @@ push-operator:
 	docker tag legion/k8s-operator:latest nexus.cc.epm.kharlamov.biz:443/legion/k8s-operator:${TAG}
```
Comment (dsuslov) [inline Makefile]: @kirillmakhonin can you avoid hardcoding our internal infrastructure urls and domains like this nexus.cc.epm.kharlamov.biz ?

File: containers/feedback-aggregator/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+WORKDIR "${LEGION_DIR}"
+CMD ["./server"]
+
+#########################################################
+#########################################################
+#########################################################
+
+FROM golang:1.11.6-alpine3.9 as tests
+
+COPY --from=builder /go/src/github.com/legion-platform/legion/legion/feedback-aggregator/server /go/src/github.com/legion-platform/legion/legion/feedback-aggregator/server
```
Comment (dsuslov) [inline containers/feedback-aggregator/Dockerfile]: @kirillmakhonin why you do not copy to subfolder dedicated to tests ?

**Review by dsuslov — CHANGES_REQUESTED** [review]
👎

File: Makefile (hunk 2 above)
Comment (kirillmakhonin) [reply Makefile]: done

**Review by mcdoker18 — CHANGES_REQUESTED** [review]
Remove the following files:
* legion/feedback-aggregator/cover.out
* legion/feedback-aggregator/build

**Comment by mcdoker18** [comment]
bot, test all

**Comment by mcdoker18** [comment]
bot, test app

**Comment by mcdoker18** [comment]
bot, test app

**Review by mcdoker18 — APPROVED** [review]

**Review by alexander-semenets — APPROVED** [review]

**Pull request merged**